required/latex-lab/latex-lab-math.dtx

```diff
@@ -997,13 +997,15 @@
 % \end{macro}
 %
 % The html reader.
+% \changes{0.6o}{2025-05-01}{set catcode of hat to other too, tagging issue 836}
 %    \begin{macrocode}
 \cs_new_protected:Npn \@@_AF_html_reader:w#1</h2>#2<p>#3</p>#4<p>#5</p>#6<math{
   \begingroup
    \char_set_catcode_other:N\{
    \char_set_catcode_other:N\}
    \char_set_catcode_other:N\#
    \char_set_catcode_other:N\%
+   \char_set_catcode_other:N\^
```

Comment:
we should probably do `\ ` as well?

Reply:
I don't think it is needed. I added a test and it works without catcode change. The char broke not because if was executed but because if was invalid.

Reply:
yes I guess so